src/main/resources/org/jenkinsci/plugins/scriptsecurity/scripts/ScriptApproval/tabs/signature_approved.jelly

```diff
@@ -0,0 +1,70 @@
+<?jelly escape-by-default='true'?>
+<j:jelly xmlns:j="jelly:core" xmlns:st="jelly:stapler">
+    <div class="custom-pane-frame-padding">
+        <div>Signatures already approved:</div>
+        <textarea readonly="readonly" id="approvedSignatures" rows="10" cols="80">
+            <j:forEach var="line" items="${it.approvedSignatures}">${line}<st:out value="&#10;"/></j:forEach>
+        </textarea>
+        
+        <div class="space-before">Signatures already approved assuming permission check:</div>
+        <textarea readonly="readonly" id="aclApprovedSignatures" rows="10" cols="80">
+            <j:forEach var="line" items="${it.aclApprovedSignatures}">${line}<st:out value="&#10;"/></j:forEach>
+        </textarea>
+        
+        <j:set var="dangerousApprovedSignatures" value="${it.dangerousApprovedSignatures}"/>
+        <j:if test="${!empty(dangerousApprovedSignatures)}">
+            <div class="space-before">Signatures already approved which <strong><font color="red">may have introduced security vulnerabilities</font></strong> (recommend clearing):</div>
+            <textarea readonly="readonly" id="dangerousApprovedSignatures" rows="10" cols="80">
+                <j:forEach var="line" items="${dangerousApprovedSignatures}">${line}<st:out value="&#10;"/></j:forEach>
+            </textarea>
+        </j:if>
+        <p>
+            You can also remove all previous signature approvals:
+            <button onclick="if (confirm('Really delete all approvals? Any existing scripts will need to be rerun and signatures reapproved.')) {clearApprovedSignatures()}">Clear Approvals</button>
```

Comment:
```suggestion
            <span class="yui-button">
                <button onclick="if (confirm('Really delete all approvals? Any existing scripts will need to be rerun and signatures reapproved.')) {clearApprovedSignatures()}">Clear Approvals</button>
            </span>
```

Reply:
signature_approved = only moved, nothing intented to be changed